pydicom/dataset.py

```diff
@@ -1047,6 +1042,43 @@ def save_as(self, filename, write_like_original=True):
 
         pydicom.dcmwrite(filename, self, write_like_original)
 
+    def ensure_file_meta(self):
+        """Create an empty file meta dataset if none exists."""
+        self.file_meta = getattr(self, 'file_meta', None) or Dataset()
+
+    def fix_meta_info(self, include_checks=True):
+        """Ensure the file meta info exists and has the correct values
```

Comment:
`include_checks` could probably be `enforce_standard` instead since that's pretty much exactly what's being done when `True`.

Reply:
Agreed - done.